addons/revolutionaries/CfgWeapons.hpp

```diff
@@ -28,4 +28,14 @@ class CfgWeapons {
             EQUIP_POINTER(acc_flashlight);
         };
     };
+    
+    class CUP_arifle_CZ805_GL;
+    class TACU_Revolutionaries_W_CZ805_GL: CUP_arifle_CZ805_GL {
+        dlc = QUOTE(PREFIX);
+        scope = 1;
+        scopeCurator = 1;
+        class LinkedItems {
+            EQUIP_POINTER(acc_flashlight);
```

Comment:
Won't work ingame.
```suggestion
            class LinkedItemSide {
                slot = "CUP_PicatinnySideMountCZ805";
                item = "acc_flashlight";
            };
```
Some older CUP weapons uses an older "slot" name, meaning the macro for EQUIP_POINTER work work.
The CZ805 only recently got updated textures/models, the configs more or less remained the same, so it still uses that old method.

For future reference, if attachments don't show up, you can find out the proper slot names ingame in the editor:
"Tools" -> "Config Viewer"
Double-click the classname of the weapon in question under `CfgWeapons`, and then double click `WeaponSlotsInfo`.
Should have all the info there.

Also, make this change to the original CZ805 you have in this file, as the flashlight in that doesn't work either.